aiplatform/api/AIPlatform.Samples/PredictTextEmbeddingsSample.cs

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START aiplatform_sdk_embedding]
+
+using Google.Cloud.AIPlatform.V1;
+using wkt = Google.Protobuf.WellKnownTypes;
+using System;
+using System.Collections.Generic;
+
+public class PredictTextEmbeddingsSample
+{
+    public int PredictTextEmbeddings(
+        string projectId = "your-project-id",
+        string locationId = "us-central1",
+        string publisher = "google",
+        string model = "textembedding-gecko@001"
+    )
+    {
+        // Initialize client that will be used to send requests.
+        // This client only needs to be created once,
+        // and can be reused for multiple requests.
+        var client = new PredictionServiceClientBuilder
+        {
+            Endpoint = $"{locationId}-aiplatform.googleapis.com"
+        }.Build();
+
+        // Configure the parent resource.
+        var endpoint = EndpointName.FromProjectLocationPublisherModel(projectId, locationId, publisher, model);
+
+        // Initialize request argument(s).
+        var instances = new List<wkt::Value>
```

Comment:
Refer to Value directly instead of referring it via wkt, Here and all other places.

Reply:
There's also another `Value` in `Google.Cloud.AIPlaform.V1`, so I can't refer to it directly, I need to quantify which one, hence `wkt`